ui/js/dfv/src/helpers/validators.js

```diff
@@ -109,8 +109,14 @@ export const numberValidator = (
 export const dateTimeValidator = (
 	yearRange,
 	momentFormat,
-) => ( value ) => {
-	if ( 'undefined' === typeof yearRange || ! yearRange.length ) {
+	allowEmpty = false,
```

Comment:
@Shelob9 I question if this should default to false. Please check where else it is used.

Reply:
```suggestion
	allowEmpty = true,
```

This should default as true.